src/sagemaker_containers/training_environment.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+from sagemaker_containers.environment import Environment
+
+
+class TrainingEnvironment(Environment):
+    """Provides access to aspects of the container environment relevant to training jobs.
+    """
+
+    @property
+    def input_dir(self):
+        """The base directory for training data and configuration files."""
+        return os.path.join(self.base_dir, 'input')
+
+    @property
+    def input_config_dir(self):
+        """The directory where standard SageMaker configuration files are located."""
+        return os.path.join(self.base_dir, 'input/config')
```

Comment:
I would rewrite this as

```python
os.path.join(self.base_dir, 'input', 'config')
```

while I think os.path.join() will figure out the / in your string correctly (even on Windows), Windows only users can be confused as to why input/config is a path.

Reply:
done